### PR TITLE
WIP: Checkout my new feature

### PR DESCRIPTION
**Pull request by adomakor**
@brysgo can you check this out

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -71,7 +71,7 @@
         return (
           <div>
             <div>
-              Hello, {this.state.name}
+              Hello, {this.state.name}! You're awesome!!
```
Comment (brysgo) [inline index.html]: Looks good!

Comment (adomakor) [inline index.html]: So it only shows lines I've changed?

Comment (brysgo) [inline index.html]: Yes but it lets you browse the whole branch if you need more context.

**Comment by brysgo** [comment]
Make sure you fix your commit message before you merge!!

**Pull request merged**